src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import './utils/variables';
+@import './blocks/calendar';
+
+* {
```

Comment:
It's a good practice to split your styles into separate files according to their purpose. For example, global styles like `* { box-sizing: border-box; }` and `.sections` could be moved to a `_global.scss` or `_reset.scss` file. This will make your code more organized and easier to maintain.